### PR TITLE
use local server - fixes #217

### LINKED ISSUE #217: Use a local server for the tests
That will speed them up considerably.

We can have one test that tests a real website, but the rest can just hit the local server.

Example: https://github.com/sindresorhus/got/blob/32d9077a7661971aca536abd9c6f8c86dd08baab/test/_server.js#L8-L24

### PR DESCRIPTION
**Pull request by SamVerschueren**
Using a local server to run the tests to speed up the tests.

- Extended `_server.js` with `createServer` and `createCookieServer`
- Added more html to `fixture.html` in order to test dom element capturing
- Extracted the cookie tests to a separate test file. Looked cleaner to me.

Feel free to provide feedback!

@sindresorhus @kevva

### CODE REVIEW

File: test/fixture.html [hunk 1]
```diff
@@ -1 +1,15 @@
-<h1>unicorns</h1>
+<html>
+<head>
+	<style>
+	#team { padding: 0px; height: 80px; width: 1024px; }
```
Comment (SamVerschueren) [inline test/fixture.html]: If this CSS line is not provided and we capture the dom `#team` element, the width and height of the output file are the size of that element, not the size we provided in the `src` function. Even when the `width` is set to `1024px`, but the padding is not set to `0px`, the output size is something like `1080px` wide. Might be related to #141.

**Comment by sindresorhus** [comment]
Awesome! The testing went from ~30s to ~20s :D Thank you Sam.

**Comment by SamVerschueren** [comment]
On my local machine it actually went to ~10s :).

**Comment by sindresorhus** [comment]
Ah yeah. ~12s without the Babel stuff.

Btw, would you like to officially become part of the team here? You've done many good PRs, so just thought I'd ask. No commitment expected.

**Comment by SamVerschueren** [comment]
> Btw, would you like to officially become part of the team here?

Would be nice :)!

**Comment by sindresorhus** [comment]
@SamVerschueren Awesome! Welcome to the team :) ✨🎉

Added you to the Pageres npm packages. Can you add yourself to the readme and the various package.json files?

**Comment by SamVerschueren** [comment]
@sindresorhus Awesome! Thank you very much :)

> Can you add yourself to the readme and the various package.json files?

Sure, no problem :)

**Comment by SamVerschueren** [comment]
Btw, @kevva is missing from the maintainers list in package.json as well?

**Comment by kevva** [comment]
No way!

**Comment by sindresorhus** [comment]
:( We must correct that!

**Comment by SamVerschueren** [comment]
Adding you as we speak :).

**Comment by sindresorhus** [comment]
@SamVerschueren Oops, I already did it.

**Pull request merged**